tools/docker/cuda11.2.2-cudnn8-ubuntu20.04-manylinux2014-multipython.Dockerfile

```diff
@@ -0,0 +1,107 @@
+# Dockerfile to build a manylinux 2010 compliant cross-compiler.
```

Comment:
You don't have to add these docker files, I have updated the CUDA11.2 docker to 11.2.2 in the new PR(mergered) of @jq.

Reply:
removed